bach/tests/functional/bach/test_df_setitem.py

```diff
@@ -17,44 +17,53 @@
     assert_equals_data, CITIES_INDEX_AND_COLUMNS, get_bt_with_railway_data, get_df_with_test_data, run_query
 
 
-def check_set_const(engine, constant, expected_series: Type[Series], expected_pg_db_type: str):
+def check_set_const(engine, constants: list, expected_series: Type[Series], expected_pg_db_type: str):
```

Comment:
just to follow consistency
```suggestion
def check_set_const(engine, constants: List[Any], expected_series: Type[Series], expected_pg_db_type: str):
```